GOVERNANCE.md

```diff
@@ -72,3 +72,10 @@ The Cloud Native Buildpacks Core Team is responsible for establishing a yearly r
 
 ## Updating Governance
 All substantive changes in Governance require a supermajority agreement by the Core Team.
+
+---
+#### Foototes:
+
+##### Supermajority:
+[supermajority]: #supermajority
+Throughout these documents, "supermajority" means 66%+. If there are 2 maintainers, both must vote. In the case of a team with one maintainer, their single vote is sufficient.
```

Comment:
I don't think the clarification about two is necessary (although the one about a single maintainer should stay).  Abstention is a tried-and-true way to prevent a majority.

Reply:
@zmackie mind making the change above?

Reply:
@hone  done